### DISCUSSION TITLE
Question About Predict values are same

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#142] [BaiduKDDCup2022]
My team is SlienceGTeam. I also met the '90.57% of predicted values are the same (0-th prediction)' problem. I do not think that predicting similar values is abnormal. For example, using the mean value of Patv in training dataset is a common baseline. It is even better than the open source GRU baseline. In recent LeaderBoard, we can find that there's not a big gap in scores because of most of time periods are hard to predict.  For the time periods that are hard to predict, using mean value to replace the model prediction is also a reasonable and robust strategy. Therefore, I think it's not necessary to reject the similar prediction.

**Comment by gengdd** [comment]
Indeed
